.github/workflows/npm-audit.yml

```diff
@@ -25,6 +26,19 @@ jobs:
     - name: npm audit (root)
       run: npm audit --production
 
+  audit-extensions:
+    runs-on: ubuntu-latest
```

Comment:
Is this git requirement? Ubuntu?

Reply:
It is just for running the `npm audit` command, nothing else, so just using what we do for the other libs.